homeassistant/components/mqtt/device_tracker.py

```diff
@@ -191,7 +191,7 @@ def location_name(self) -> str | None:
         return self._location_name
 
     @property
-    def source_type(self) -> SourceType | str:
+    def source_type(self) -> SourceType:
```

Comment:
This change is moved to #126594 I believe.

Reply:
> This change is moved to #126594 I believe.

You're right - reverted